beta/tests/tensorflow/data/ac_configs/resnet50_int8_w_sym_t_a_sym_t_sparsity_50.yml

```diff
@@ -0,0 +1,19 @@
+models:
+  - name: resnet50_int8_w_sym_t_a_sym_t_sparsity_50
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter: classification
+    datasets:
+      - name: imagenet_1000_classes
+        data_source: ImageNet
+        annotation: imagenet1000.pickle
+        reader: pillow_imread
```

Comment:
```suggestion
```
let's try to unify ac configs. Seems like it affect total accuracy. Do it here and elsewhere this snippet appear